docs/references/kontena-yml.md

```diff
@@ -314,12 +349,12 @@ db:
 
 #### external_links
 
-Link to services in the same grid outside application scope. `external_links` follow semantics similar to links.
+Link to services in the same grid outside stack scope. `external_links` follow semantics similar to links. However you have to use `<stack>/<service>` format instead:
 
 ```
 external_links:
-  - loadbalancer
-  - common-redis:redis   
+  - common/loadbalancer
+  - common/redis:redis   
 ```
 
 #### links
```

Comment:
Links are also used for LB registration, by linking to the lb service. The linked LB service can also be outside of this stack, using `links: - lbstack/lbservice`.